app/src/main/res/layout/gpodnetauth_credentials.xml

```diff
@@ -33,7 +33,11 @@
         android:layout_margin="8dp"
         android:focusable="true"
         android:focusableInTouchMode="true"
-        android:cursorVisible="true"/>
+        android:cursorVisible="true"
+        android:maxLines="1"
```

Comment:
that's why one also sets the `inputType` to "text", implicitly meaning that the content is single line only.